api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceServiceImpl.java

```diff
@@ -102,15 +111,46 @@ public void updateUserRoles(String ns, String id, Set<WorkspaceUserRole> userRol
         currentUserRole.setRole(mapValue.getRole());
         userRoleMap.remove(currentUserRole.getUser().getUserId());
       } else {
+        WorkspaceACLUpdate removedUser = new WorkspaceACLUpdate();
+        removedUser.setEmail(currentUserRole.getUser().getEmail());
+        removedUser.setCanCompute(false);
+        removedUser.setCanShare(false);
+        removedUser.setAccessLevel("NO ACCESS");
```

Comment:
I would make the enum values in WorkspaceAccessLevel named exactly the strings we want, and use toString() on those enum values throughout when dealing with FC access levels.

Reply:
The one thing here is that we aren't actually representing `NO ACCESS` in our DB, unless we want that to be added to the enum and just be unused except here? (I think that's fine? Except that they are looking for spaces in their string, which I don't know if you can do? I can check that.)

Reply:
OK. Let's just define a separate set of constants?

Reply:
You totally can, moving over to that.